content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+name is added, the :guilabel:`Sales Order Item` field will automatically populate with the most
+recent prepaid sales order item that has time remaining.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Click on the :guilabel:`Timesheets` tab on the :guilabel:`Ticket`. To track the time spent on this
+ticket against the hours sold on the :abbr:`SO (sales order)`, click :guilabel:`Add a line`. Choose
+an :guilabel:`Employee`, and enter the number of :guilabel:`Hours Spent`. As new lines are added to
+tab, the :guilabel:`Remaining Hours on SO` is updated.
+
+.. image:: track_and_bill/track-bill-remaining-hours-total.png
+   :align: center
+   :alt: View of the timesheets tab on a ticket with an emphasis on the remaining hours on an SO.
+
+.. note::
+   If the number of hours on the :guilabel:`Timesheets` tab goes over the number of hours sold, the
+   :guilabel:`Remaining Hours of SO` will turn red.
+
+As hours are added to the :guilabel:`Timesheets` tab, they are updated in the :guilabel:`Delivered`
+field on the :guilabel:`Sales Order`.
+
+Invoice post-paid support services
+==================================
+
+When support services are billed based on the amount of time spent on an issue, an invoice cannot
+the total number of hours required to solve the problem have been entered on a timesheet. In this
+case, a service product with the invoicing policy set to :guilabel:`Based on Timesheets` (like the
+one created above) would be used.
+
+Create a sales order with a time tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+support services product. To do this, go to :guilabel:`Odoo Dashboard --> Sales --> Orders -->
+Quotations --> New`.
+
+Fill out the :guilabel:`Quotation` with the :guilabel:`Customer` information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the post-paid services
+product configured in the steps above. After updating any other necessary information,
+:guilabel:`Confirm` the quotation.
+
+.. note::
+   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at
+   this time. (Since no services have been performed, nothing has been *delivered*, therefore there
+   there is nothing to invoice).
+
+Create a helpdesk ticket for time-tracked services
+--------------------------------------------------
+
+To record a :guilabel:`Timesheet` entry, go to the :menuselection:`Helpdesk` app and select the
+appropriate :guilabel:`Team`.
+
+If there is already an existing ticket for this issue, select it from the kanban view to go to the
+details form. If there is no existing ticket for this customer issue, click :guilabel:`New` to
+create a new ticket and enter the necessary customer information.
+
+In the :guilabel:`Sales Order Line` drop-down, select the :abbr:`SO (sales order)` created in the
+previous step.
```

Comment:
In the :guilabel:`Sales Order Line` drop-down menu, on the ticket detail form, select the :abbr:`SO (Sales Order)` that was created in the previous step.